sdk/react/src/shared/ItemSelectionScrollView/ItemSelectionList/ItemSelectionList.styles.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+  column: {
+    //
```

Comment:
do we really need these empty styles?

Reply:
Those are used by https://github.com/ooyala/native-skin/pull/879/files#diff-6a62de6d02c74a360ae9ff8b478c802aR26 but they were lost. I think someone who wrote that line had something in mind...

Reply:
It can be just copy/paste. Anyway, if we don't use them let's remove the dead code

Reply:
Removed.